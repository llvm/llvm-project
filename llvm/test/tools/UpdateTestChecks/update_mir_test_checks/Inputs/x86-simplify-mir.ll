; RUN: llc < %s -mtriple=x86_64 -stop-after=finalize-isel -simplify-mir | FileCheck %s

define void @f() {
  ret void
}
