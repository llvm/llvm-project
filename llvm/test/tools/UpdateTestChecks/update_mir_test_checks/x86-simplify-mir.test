# REQUIRES: x86-registered-target
## Check that update_mir_test_checks handles -simplify-mir

# RUN: cp -f %S/Inputs/x86-simplify-mir.ll %t.ll && %update_mir_test_checks %t.ll
# RUN: diff -u %S/Inputs/x86-simplify-mir.ll.expected %t.ll
