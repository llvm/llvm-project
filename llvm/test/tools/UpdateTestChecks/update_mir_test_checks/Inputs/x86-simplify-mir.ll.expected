; NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
; RUN: llc < %s -mtriple=x86_64 -stop-after=finalize-isel -simplify-mir | FileCheck %s

define void @f() {
  ; CHECK-LABEL: name: f
  ; CHECK: bb.0 (%ir-block.0):
  ; CHECK-NEXT:   RET 0
  ret void
}
